tests/config/config.d/zookeeper.xml

```diff
@@ -1,5 +1,9 @@
 <yandex>
     <zookeeper>
-        <implementation>testkeeper</implementation>
+        <implementation>etcdkeeper</implementation>
```

Comment:
probably we need to change it from test environment to check different zookeeper alternatives in tests?